clang/lib/Headers/devicelib/__devicelib.h

```diff
@@ -0,0 +1,51 @@
+//==------------ devicelib.h - macros for devicelib wrappers ---------------==//
```

Comment:
This path (i.e. `devicelib/`) doesn't conflict with anything else, does it?

Reply:
Not sure what do you mean by a "conflict". When installed on a system, this directory resides in `$prefix/lib/clang/$version/` directory.

Reply:
OK, then.